### ISSUE TITLE
Publish code coverage : Task Fails if SummaryFile does not exist

### ISSUE THREAD

**Issue by chrish619** [Microsoft/vsts-tasks#4859]
Task fails, even when "Fail when Code Coverage Results Are Missing" is unchecked.

File Glob searches are also in use.

```
2017-07-21T20:36:34.9146388Z ##[error]Unable to process command '##vso[codecoverage.publish codecoveragetool=Cobertura;]' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
2017-07-21T20:36:34.9156390Z ##[error]'SummaryFile' has to be specified.
```
![image](https://user-images.githubusercontent.com/2427358/28517892-efa520ac-705d-11e7-9e1e-8e088fea93c2.png)

**Comment by lkillgore** [comment]
Hi @chrish619, sorry I haven't gotten back to you sooner.  If I'm not mistaken, this is a duplicate of https://github.com/Microsoft/vsts-tasks/issues/4800, which was fixed in 1.121.0.  I'm going to close this ticket with the expectation that's the case.  If I'm wrong, please let me know.  Thanks!

**Comment by madhurig** [comment]
Closing since we haven't had any activity on this for several months and this is as the task was designed.
